tests/data/nplinker_demo1.toml

```diff
@@ -1,9 +1,3 @@
-loglevel = "DEBUG"
-logfile = ""
-repro_file = ""
-[dataset]
-root = "platform:MSV000079284"
-platform_id = "MSV000079284"
-
-[webapp]
-tables_metcalf_score = 3.0
+root_dir = "@format {env[NPLINKER_ROOT_DIR]}"
```

Comment:
What kind of syntax is the one in the string? Does it work?

Reply:
It's [Dynaconf syntax](https://www.dynaconf.com/dynamic/#format-token) to specify dynamic variable. `{env[NPLINKER_ROOT_DIR]}` means the value will be taken from the environment variable "NPLINKER_ROOT_DIR" defined in the `tests/conftest.py`. 
Why setting a dynamic variable here? Because we have to make sure the given root dir exist, so better to create a temporary dir for it (in the `tests/conftest.py`).